### PR TITLE
[12.0][IMP] hr_timesheet_task_required: note about multi-company setting

### PR DESCRIPTION
**Pull request by alexey-pelykh**
Added icon

### CODE REVIEW

File: hr_timesheet_task_required/views/res_config_settings.xml [hunk 1]
```diff
@@ -17,6 +17,7 @@ License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
                     </div>
                     <div class="o_setting_right_pane">
                         <label for="is_timesheet_task_required"/>
+                        <span class="fa fa-lg fa-building-o" title="Values set here are company-specific." role="img" aria-label="Values set here are company-specific." groups="base.group_multi_company"/>
```
Comment (pedrobaeza) [inline hr_timesheet_task_required/views/res_config_settings.xml:20]: Put this only if the group multi-company is present

Comment (alexey-pelykh) [reply hr_timesheet_task_required/views/res_config_settings.xml:20]: Check the very end of the line - is that what needed?

Comment (pedrobaeza) [reply hr_timesheet_task_required/views/res_config_settings.xml:20]: Yes! That's the problem with long lines here in GitHub, that are difficult to be seen...

**Review by pedrobaeza — APPROVED** [review]

File: hr_timesheet_task_required/views/res_config_settings.xml (hunk 1 above)
Comment (alexey-pelykh) [reply hr_timesheet_task_required/views/res_config_settings.xml:20]: Yeah, true!

**Review by astirpe — APPROVED** [review]
Tested locally

**Pull request merged**